packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -487,7 +487,9 @@ export class EpochTracker implements IPersistedFileCache {
 	}
 
 	private fileEntryFromEntry(entry: IEntry): ICacheEntry {
-		return { ...entry, file: this.fileEntry };
+		const maybeCacheEntry = entry as ICacheEntry;
```

Comment:
you probably also want to be more careful here, like do an check for the property `"file" in entry`